imneversorry.py

```diff
@@ -17,16 +18,20 @@
 
 # Add valid command line arguments
 arg_parser = ArgumentParser()
-arg_parser.add_argument('--verbose', help='Enable verbose logging for debugging.', action='store_true')
+arg_parser.add_argument(
+    '--verbose', help='Enable verbose logging for debugging.', action='store_true')
 args = arg_parser.parse_args()
 
 if args.verbose:
-    logging.basicConfig(format="%(levelname)s: %(message)s", level=logging.DEBUG)
+    logging.basicConfig(format="%(levelname)s: %(message)s",
+                        level=logging.DEBUG)
     logging.info('Verbose ĺogging enabled')
 
 cfg = ConfigParser()
 cfg.read('env.cfg')
 
+BANNED_CHANNELS = ast.literal_eval(cfg['TELEGRAM']['banned_channels'])
```

Comment:
I am just wondering why you use `ast.literal_eval` here. Doesn't configparser support type conversion?

Reply:
It apparently doesn't, it just gave a string like "[1,2,3]" which had to be parsed somehow.

`ast.literal_eval` should be safer than just `eval` for parsing purposes.